client/app/telecom/pack/slots/emailPro/detail/pack-emailPro-detail.html

```diff
@@ -0,0 +1,139 @@
+<section class="telecom-section">
+    <div class="telecom-section-header">
+        <header role="banner">
+            <section-back-link section-back-link-to-state="telecom.pack"
+                               section-back-link-title="{{ $ctrl.pack.description || $ctrl.pack.packName }}">
+            </section-back-link>
+            <h1 data-translate="email_pro_detail_title"
+                data-translate-values="{name: $ctrl.serviceName}"></h1>
```

Comment:
`{ name: $ctrl.serviceName }` (missing spaces after opening bracket and closing one)